drag_and_drop_v2/drag_and_drop_v2.py

```diff
@@ -233,8 +236,18 @@ def studio_view(self, context):
         for js_url in js_urls:
             fragment.add_javascript_url(self.runtime.local_resource_url(self, js_url))
 
+        # Do a bit of manipulation so we get the appearance of a list of zone options on
+        # items that still have just a single zone stored
+        data = copy.deepcopy(self.data)
```

Comment:
@haikuginger 
> making the change completely silently just because the Studio view was opened

Was that actually happening? I don't remember off the top of my head whether an XBlock write to a `Scope.content` field from a Studio `student_view` has any effect or not; there's a chance it does not. (Studio uses a different runtime for previews vs. XBlock edits. And the LMS student_view definitely cannot modify Scope.content fields.)